src/sonic-yang-models/yang-models/sonic-vxlan.yang

```diff
@@ -0,0 +1,121 @@
+module sonic-vxlan {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-vxlan";
+    prefix svxlan;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+    import sonic-vlan {
```

Comment:
The tests are failing in Sonic-utils specifically GCU because of yang known issue: https://github.com/Azure/sonic-buildimage/issues/9312

Please comment out `sonic-vlan` import and add a note 
```
    // Comment sonic-vlan import here until libyang back-links issue is resolved for VLAN leaf reference.
    //import sonic-vlan {
    //    prefix vlan;
    //}
```